src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,45 @@
+package ui;
+
+/**
+ * Deaks with interactions with the user
```

Comment:
Small typo here

Reply:
Thank you for spotting the mistake there! 😄